ui/components/app/modals/account-details-modal/account-details-modal.component.js

```diff
@@ -22,6 +29,39 @@ export default class AccountDetailsModal extends Component {
     trackEvent: PropTypes.func,
   };
 
+  state = {
+    name: this.props.selectedIdentity.name || '',
+  };
+
+  componentDidMount() {
```

Comment:
[req] Rather than extending this component to rely more on class component lifecycle methods, lets convert this component to a functional one. This component is relatively easy to do because it only currently has a render method and no state.